modules/oe_whitelabel_helper/oe_whitelabel_helper.info.yml

```diff
@@ -5,7 +5,13 @@ package: OpenEuropa Whitelabel Theme
 core_version_requirement: ^9.2
 
 dependencies:
+  - drupal:node
   - oe_bootstrap_theme:oe_bootstrap_theme_helper
   - openeuropa:oe_corporate_blocks
 
 'interface translation project': oe_whitelabel_helper
+
+config_devel:
+  optional:
+    # A view mode should not depend on a theme, so it is exported in a module.
```

Comment:
needles to say, as in not needed, lets add comments only where clarifications are needed

Reply:
I have a habit to add a comment whenever I was not sure about something myself and had to look it up or learn it.

Reply:
But yeah let's drop, otherwise we will have comments like this all over the place.